docs/api/api/api.txt

```diff
@@ -1933,6 +1933,15 @@ XmlResult: required_checks
 == Staging
   WARNING: This API is currently in beta and unstable
 
+=== Staging Workflow
+
+<staging_workflow_project>: The project for which we want to create the staging workflow
+
+POST /staging/<staging_workflow_project>/workflow
```

Comment:
You forgot to mention what it does, and I would move this into the 'Staging Projects' section - or rename that section, but don't make it 2.